### PR TITLE
[v6r17] SetupSite to include NoSQL parameters

### PR DESCRIPTION
**Pull request by fstagni**
And several other fixes

### CODE REVIEW

File: TransformationSystem/Client/TaskManager.py [hunk 1]
```diff
@@ -337,7 +338,7 @@ def getSubmittedTaskStatus( self, taskDicts ):
 
       newStatus = self.requestClient.getRequestStatus( taskDict['ExternalID'] )
       if not newStatus['OK']:
-        log = self._logVerbose if 'not exist' in newStatus['Message'] else self.__logWarn
+        log = self._logVerbose if 'not exist' in newStatus['Message'] else self._logWarn
```
Comment (phicharp) [inline TransformationSystem/Client/TaskManager.py]: Thanks... Grrrr.... I don't understand why the code checker didn't pick up that typo?

**Comment by fstagni** [comment]
Review anyone?

File: FrameworkSystem/Client/ComponentInstaller.py [hunk 2]
```diff
@@ -195,40 +196,41 @@ def loadDiracCfg( self, verbose = False ):
...
       gLogger.notice( 'Reading Root MySQL Password from local configuration' )
+    else:
+      gLogger.warn( 'MySQL root password not found' )
 
-    self.mysqlUser = self.localCfg.getOption( cfgInstallPath( 'Database', 'User' ), '' )
-    if self.mysqlUser:
-      if verbose:
-        gLogger.notice( 'Reading MySQL User from local configuration' )
+    self.mysqlUser = self.localCfg.getOption( cfgInstallPath( 'Database', 'User' ), self.mysqlUser )
+    if verbose and self.mysqlUser:
```
Comment (chaen) [inline FrameworkSystem/Client/ComponentInstaller.py]: I don't understand why do we use this verbose flag instead of setting the gLogger to an appropriate level ?

File: tests/Jenkins/install.cfg [hunk 3]
```diff
@@ -63,8 +63,6 @@ LocalInstallation
   {
     User = VAR_NoSQLDB_User
     Password = VAR_NoSQLDB_Password
-    RootUser = VAR_NoSQLDB_RootUser
```
Comment (chaen) [inline tests/Jenkins/install.cfg]: Why is this removed ?

File: FrameworkSystem/Client/ComponentInstaller.py (hunk 2 above)
Comment (fstagni) [inline FrameworkSystem/Client/ComponentInstaller.py]: IIUC it was done so that depending on the caller, different levels could be set. But I agree it's a bit weird way of working. I don't want to change this code right now as it's quite a pain to test it, so not for this release.

**Comment by chaen** [comment]
Review OK

File: FrameworkSystem/Client/ComponentInstaller.py (hunk 2 above)
Comment (JanEbbing) [inline FrameworkSystem/Client/ComponentInstaller.py]: But now if `verbose` is `False` it will print out a bunch of warnings that are potentially misleading?
